### PR TITLE
Fixes the slime blueprints counting for the blueprints objective

### PR DESCRIPTION
**Pull request by Time-Green**
Fixes https://github.com/yogstation13/yogstation/issues/726

### CODE REVIEW

File: code/game/gamemodes/objective_items.dm [hunk 1]
```diff
@@ -118,7 +118,8 @@
 
 /datum/objective_item/steal/blueprints/check_special_completion(obj/item/I)
 	if(istype(I, /obj/item/areaeditor/blueprints))
-		return 1
+		if(I.type == /obj/item/areaeditor/blueprints)
+			return 1
```
Comment (X-TheDark) [inline code/game/gamemodes/objective_items.dm]: Just replace line 120 with 121.

Comment (Time-Green) [reply code/game/gamemodes/objective_items.dm]: Sure, I'll go right ahead.
Soon.

**Comment by Time-Green** [comment]
Done.

**Comment by Super3222** [comment]
Thank you.

**Pull request merged**